main.js

```diff
@@ -30,15 +30,16 @@ function drawNumberline(){
   line.attr('stroke-width', '3');
 
   var circleY = lineStart.y;
-  var textY = circleY - 8;
-  var descY = textY + hoursFontSize;
+  var textY = circleY - 1.5;
+  var descY = textY - hoursFontSize - 50;
```

Comment:
The `50` here is dependent on other values - circle radius, etc. - so it'd be nice to compute it from them. For instance if I change the canvas size in the CSS and increase the circle radius, then this might fall halfway inside the circle.